master/pkg/device/device.go

```diff
@@ -36,9 +36,12 @@ func (t Type) Proto() devicev1.Type {
 	}
 }
 
+// ID the type of Device.ID.
+type ID = int
```

Comment:
```suggestion
type ID int
```

Reply:
that has added a few unfortunate casts here and there but overall I think it looks nicer

Reply:
i think it's strictly better usually, e.g., https://go.dev/play/p/ftp_m7N-wzm